### PR TITLE
Single Quotes

### PR DESCRIPTION
**Pull request by KyGost**
Relates to #76

### CODE REVIEW

**Comment by KyGost** [comment]
Note: bug not enhancement
(Though I suppose they have arbitrary differences)

File: src/data/value.rs [hunk 1]
```diff
@@ -288,6 +288,7 @@ impl Value {
             (DataType::Boolean, AstValue::Boolean(v)) => {
                 Ok(nullable.into_value(Value::OptBool(Some(*v)), Value::Bool(*v)))
             }
+            (DataType::Text, AstValue::SingleQuotedString(v)) => Ok(Value::Str(v.to_string())),
```
Comment (KyGost) [inline src/data/value.rs:291]: I noticed after the fact that others have solved this issue too.
Though both of them changed it to
`Ok(nullable.as_value(Value::OptStr(Some(v.clone())), Value::Str(v.clone())))`

Would that be better?

Comment (panarch) [reply src/data/value.rs:291]: Oh yes, that should be changed like you wrote :) thanks

**Review by panarch — CHANGES_REQUESTED** [review]

File: src/data/value.rs (hunk 1 above)
Comment (KyGost) [reply src/data/value.rs:291]: Done :-)

**Review by panarch — APPROVED** [review]
Thanks 👍 👍 👍

**Pull request merged**